R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
```

Comment:
Yes the lower bounds are weird. 
> and then check for initial.window.size > 0 (which you have to do anyways).

Should I test whether the initial window (in `instatiateResampleInstance`) has at least one observation in it? Having a value of 0 in the lower bound would be odd, how about a lower bound of the machines floating point precision? (though this is a bit odd I think it's the 'most correct' thing)